js/story.js

```diff
@@ -0,0 +1,44 @@
+(function() {
+  // make IE9+ support forEach:
+  if (window.NodeList && !NodeList.prototype.forEach) 
+    NodeList.prototype.forEach = Array.prototype.forEach;
+
+  document.querySelector('.stories__story-image').classList.add('stories-image__animate');
+  document.querySelector('.stories__story-title-small').classList.add('stories-title-small__animate');
+  document.querySelector('.stories__story-title').classList.add('stories-title__animate');
+  document.querySelector('.stories__story-text').classList.add('stories-text__animate');
+  
+  window.addEventListener('scroll', animateStory);
```

Comment:
Je kunt deze listener het best [throttlen](https://css-tricks.com/debouncing-throttling-explained-examples/), of gebruik maken van requestAnimationFrame, want dit zorgt voor slechte performance, vooral op oudere devices en (iets) oudere mobiele telefoons.

Reply:
@hidde +1 Heb gekozen voor de requestAnimationFrame oplossing. Dit werk ook goed voor scroll events. En hebben dan ook geen extra third party / extra functions nodig.